src/js/components/Nav/Nav.tsx

```diff
@@ -29,4 +29,10 @@ const Nav: React.FC<Props> = ({
   return <Tag {...other} className={classes} />;
 };
 
-export default Nav;
+const NavWithWrapper: React.FC<Props> = props => (
+  <div className="overflow-hidden">
+    <Nav {...props} />
+  </div>
+);
+
+export default NavWithWrapper;
```

Comment:
Needed the overflow hidden to hide the shadow on the bottom side. But wanted the option to only import the Nav if necessary (easier for tests aswell)